include/prereq.mk

```diff
@@ -63,6 +63,14 @@ define RequireHeader
   $$(eval $$(call Require,$(1),$(2)))
 endef
 
+define RequireCHeader
+  define Require/$(1)
+    printf '#include <$(1)>\n int main(){}' | gcc -x c -
```

Comment:
Looks like this may write a.out to `pwd`?
For the record: prereq-build.mk uses this cmdline: `gcc -include ncurses.h -x c -o $(TMP_DIR)/a.out`

Reply:
Oh thanks! Think a better solution would be generalize the ncurses check

Reply:
And yes i notice the a.out but i tought it was part of me testing command.